rpcd/playbooks/roles/rpc_support/tasks/raid_install.yml

```diff
@@ -15,11 +15,11 @@
 
 - name: Install HWRaid packages
   apt:
-    name: "{{ item }}"
+    deb: "{{ item }}"
     state: present
     update_cache: yes
     cache_valid_time: 600
-  with_items: "{{ hwraid_apt_packages }}"
+  with_items: "{{ hwraid_apt_package_urls }}"
   when: hwraid_apt_packages is defined
```

Comment:
I guess that hwraid_apt_packages is now only used to trigger the when, so we can probably remove the hwraid_apt_packages from the defaults, remove the when condition, and change the with_items: by

`with_items: "{{ hwraid_apt_package_urls | default([]) }}"`